src/modes/linear_world.cpp

```diff
@@ -350,10 +350,13 @@ void LinearWorld::newLap(unsigned int kart_index)
 
         std::string s = StringUtils::timeToString(time_per_lap);
 
-        irr::core::stringw m_fastest_lap_message;
+        // Store the temporary string because clang would mess this up
+        // (remove the stringw before the wchar_t* is used).
+        core::stringw css = kart->getName();
+
         //I18N: as in "fastest lap: 60 seconds by Wilber"
-        m_fastest_lap_message += _C("fastest_lap", "%s by %s", s.c_str(),
-                                    core::stringw(kart->getName()));
```

Comment:
I noticed a few things we should have tried first:
1) can you try removing the core::stringw, i.e. just leaving
kart->getName()
2) if this doesn't work, what about kart->getName().c_str()
?